app/src/main/java/com/nextcloud/talk/models/json/chat/ChatUtils.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Nextcloud Talk application
+ *
+ * @author Mario Danic
+ * Copyright (C) 2017-2018 Mario Danic <mario@lovelyhq.com>
+ * @author Marcel Hibbe
+ * Copyright (C) 2021 Marcel Hibbe <dev@mhibbe.de>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.nextcloud.talk.models.json.chat
+
+import java.util.*
+
+class ChatUtils {
+    companion object {
+        fun getParsedMessage(message: String?, messageParameters: HashMap<String?, HashMap<String?, String?>>?):
+                String? {
+            var message = message
+            if (messageParameters != null && messageParameters.size > 0) {
+                for (key in messageParameters.keys) {
+                    val individualHashMap = messageParameters[key]
+                    val type = individualHashMap?.get("type")
+                    if (type == "user" || type == "guest" || type == "call") {
+                        message = message?.replace("{$key}", "@" + individualHashMap["name"])
+                    } else if (individualHashMap?.containsKey("link") == true) {
```

Comment:
== true is unnecessary.

Reply:
![grafik](https://user-images.githubusercontent.com/2932790/109212339-af5caa80-77af-11eb-9146-e0de7a90e3e2.png)

Reply:
True 🤣
I overlooked "?."